src/main/java/org/opensrp/config/RabbitMQConfig.java

```diff
@@ -0,0 +1,139 @@
+package org.opensrp.config;
+
+import org.opensrp.queue.RabbitMQReceiver;
+import org.smartregister.pathevaluator.plan.PlanEvaluator;
+import org.springframework.amqp.core.*;
+import org.springframework.amqp.rabbit.annotation.EnableRabbit;
+import org.springframework.amqp.rabbit.config.SimpleRabbitListenerContainerFactory;
+import org.springframework.amqp.rabbit.connection.CachingConnectionFactory;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.core.RabbitAdmin;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.amqp.rabbit.listener.MessageListenerContainer;
+import org.springframework.amqp.rabbit.listener.SimpleMessageListenerContainer;
+import org.springframework.amqp.rabbit.listener.adapter.MessageListenerAdapter;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@EnableRabbit
+@Configuration
+@ComponentScan("org.opensrp")
```

Comment:
Why do we have this?

Reply:
@rehammuzzamil remove this scanning of all classes

Reply:
It has been fixed in the new PR created for extension of Queuing https://github.com/OpenSRP/opensrp-server-core/pull/296/files  . Do I need to update it here too?